pkg/network/protocols/http/telemetry.go

```diff
@@ -63,7 +66,10 @@ func NewTelemetry(protocol string) *Telemetry {
 
 		// these metrics are also exported as statsd metrics
 		totalHitsPlain:         metricGroup.NewCounter("total_hits", "encrypted:false", libtelemetry.OptStatsd, libtelemetry.OptPayloadTelemetry),
-		totalHitsEncrypted:     metricGroup.NewCounter("total_hits", "encrypted:true", libtelemetry.OptStatsd, libtelemetry.OptPayloadTelemetry),
+		totalHitsGnuTLS:        metricGroup.NewCounter("total_hits", "encrypted:true", "tls_library:gnutls", libtelemetry.OptStatsd, libtelemetry.OptPayloadTelemetry),
+		totalHitsOpenSLL:       metricGroup.NewCounter("total_hits", "encrypted:true", "tls_library:openssl", libtelemetry.OptStatsd, libtelemetry.OptPayloadTelemetry),
+		totalHitsJavaTLS:       metricGroup.NewCounter("total_hits", "encrypted:true", "tls_library:java", libtelemetry.OptStatsd, libtelemetry.OptPayloadTelemetry),
+		totalHitsGoTLS:         metricGroup.NewCounter("total_hits", "encrypted:true", "tls_library:go", libtelemetry.OptStatsd, libtelemetry.OptPayloadTelemetry),
```

Comment:
I think we should extend (can be in a different PR) the other metrics as well to have tls tag

Reply:
Agreed:
https://datadoghq.atlassian.net/browse/USMON-618